content/docs/guides/neon-authorize-drizzle.md

```diff
@@ -0,0 +1,199 @@
+---
+title: Simplify RLS with Drizzle
+subtitle: Use Drizzle crudPolicy to manage Row-Level Security with Neon Authorize
+enableTableOfContents: true
+---
+
+<InfoBlock>
+<DocsList title="What you'll learn">
+<p>How to simplify Row-Level Security using crudPolicy</p>
+<p>Common RLS patterns with Drizzle</p>
+</DocsList>
+
+<DocsList title="Related docs" theme="docs">
+  <a href="/docs/guides/neon-authorize">About Neon Authorize</a>
+  <a href="https://orm.drizzle.team/docs/rls">RLS in Drizzle</a>
+</DocsList>
+
+</InfoBlock>
+
+## Why simplify RLS policies?
+
+Row-Level Security (RLS) is an important last line of defense for protecting your data at the database level. However, implementing RLS requires writing and maintaining separate SQL policies for each CRUD operation (Create, Read, Update, Delete), which can be both tedious and error-prone.
+
+### For example
+
+To illustrate, let's consider a simple **Todo** list app with RLS policies applied to a `todos` table. Postgres RLS policies use two types of conditions:
+
+- `USING` clause — controls which existing rows can be accessed
+- `WITH CHECK` clause — controls what new or modified data can be written
+
+Here's how these clauses apply to each operation:
```

Comment:
I find this table a little hard to understand. Maybe an explantion of `auth.user_id() = user_id` and why it's shown in the table would help here?

Reply:
Added an admonition with link to the explanation here: https://neon.tech/docs/guides/neon-authorize#what-neon-does-for-you